### ISSUE TITLE
Tensorflow 2.10.0 conflicts with jupyterlab 4.0.0

### ISSUE THREAD

**Issue by FlorianFasmeyer** [tensorflow/tensorflow#60615]
<details><summary>Click to expand!</summary> 
 
 ### Issue Type

Build/Install

### Have you reproduced the bug with TF nightly?

No

### Source

source

### Tensorflow Version

2.10.0

### Custom Code

No

### OS Platform and Distribution

Windows 10

### Python version

3.10.1

### Current Behaviour?

Using Conda:
1. Install jupyterlab
2. Run jupyterlab. Notice it works.
3. Install TensorFlow
4. Run jupyterlab: Notice it doesn't start, and an error message pop-up; The c_types package is missing.

Uninstalling Tensorflow fixes the problem, allowing jupyterlab to run again.
Related: https://github.com/jupyterlab/jupyterlab/issues/14558

### Standalone code to reproduce the issue

```shell
conda create --name my_env
activate my_env

conda Install jupyterlab=4.0.0
jupyter lab

conda install TensorFlow=2.10.0
jupyter lab
```

### Relevant log output

```shell
jupyter lab ImportError: DLL load failed while importing _ctypes: ...
```
</details>

**Comment by FlorianFasmeyer** [comment]
**Temporary solution:** install jupyterlab=3.6.3 instead of jupyterlab=4.0.0

**Comment by FlorianFasmeyer** [comment]
After four reinstalls with conda, TF 2.10.0 and jupyterlab 4.0.0, with no other packages added, always in a new environment, I finally gave up. To kill time, I replaced conda with mamba.

Solution(?): After installing mamba, using the default channels, I installed TF 2.10.0 and jupyterlab 4.0.0 (no other packages added) in a new environment, same procedure as the previous four times. And now jupyterlab works. The channel order might have had something to do with the issue, but I can't confirm.
